index.html

```diff
@@ -122,6 +139,302 @@ <h5 class="color-purple-mountains-majesty" localization="why-myriad-distribution
+            <div class="col-xs-12 col-sm-3 text-left">
+              <div class="info-group last">
+                <h5 localization="wallets-other-title">Reddit 3.7k</h5>
+
+                 <div class="platform">
+					<a href="https://www.reddit.com/r/myriadcoin/"><img src="images/cy.jpg" style="width:100%; height:100%" ></a>
+					</div>
+
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+<!-- End code for new members page >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
+
+
+<!-- new code for Merchants page >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> -->
+
+
+
+<div id="Merchants" class="section section-3 row">
+    <div class="col-xs-10 col-xs-push-1 col-sm-8 col-sm-push-2 col-md-6 col-md-push-3 text-center">
+      <h1 class="title" localization="wallets-section-header">Merchants - Tools</h1>
+    </div>
+    <div class="col-xs-12 col-sm-10 col-sm-push-1">
+      <div class="panel">
+        <div class="panel-body">
+          <div class="row">
+            <div class="col-xs-12 col-sm-3">
+              <div class="info-group text-left">
+ <div class="platform">
+  <b>  Drapis.com</b> :
+<p> &nbsp </p>			   
+	<a href="https://www.drapis.com/"><img src="images/drapis.jpg" style="width:100%; height:100%" ></a>
+	<p> &nbsp </p>			   
+	<p> &nbsp </p>			   
+	 <b>  Star Vape Wervik (Belgium)</b> :
+<p> &nbsp </p>			   
+	<a href=""><img src="images/star vape.jpg" style="width:100%; height:100%" ></a>
+<p> &nbsp </p>			   
+<p> &nbsp </p>			   
+ <b>  Coinbomb Game</b> :
+<p> &nbsp </p>			   
+	<a href="https://cryptap.us/myr/dice/#coinbomb"><img src="images/trader7.jpg" style="width:100%; height:100%" ></a>
+<p> &nbsp </p>			   
+	<p> &nbsp </p>			   
+	 <b>  Web Wallet</b> :
+<p> &nbsp </p>			   
+	<a href="https://jswallet.github.io/"><img src="images/trader 12.jpg" style="width:100%; height:100%" ></a>
+                       </div>
+              </div>
+            </div>
+            <div class="col-xs-12 col-sm-3 text-left">
+              <div class="info-group">
+                 <div class="platform">
+<p>
+<b>thecryptobill.com</b> : 
+<p> &nbsp </p>
+	<a href="https://thecryptobill.com/"><img src="images/trader 1.jpg" style="width:100%; height:100%" ></a>
+<p> &nbsp </p>			   
+	<p> &nbsp </p>			   
+<b>  StreamLabs</b> :
+<p> &nbsp </p>			   
+	<a href="http://cryptips.org/"><img src="images/trader5.jpg" style="width:100%; height:100%" ></a>
+<p> &nbsp </p>			   
+	<p> &nbsp </p>			   
+<b> Coinid App</b> :
+<p> &nbsp </p>			   
+	<a href="https://coinid.org/"><img src="images/trader8.jpg" style="width:100%; height:100%" ></a>
+<p> &nbsp </p>			   
+		<p> &nbsp </p>	
+	 <b> Magnum</b> :
+<p> &nbsp </p>			   
+	<a href="https://magnumwallet.co/"><img src="images/trader 13.jpg" style="width:100%; height:100%" ></a>
```

Comment:
Regarding magnumwallet, it is my understanding that it is a custodial wallet and not open source, correct? Perhaps it belongs in the "exchanges" section? In the past we've had a hard requirement that all wallets listed on myriadcoin.org **must** be open source, I'm a leaning firmly in that direction. Not sure about any of the other items here, coinid.org is good, any site that doesn't store or have access to users private keys is fine too.

Reply:
deleted maguymwallet